### PR TITLE
Iss 195 input snapping, splitting geometries

### LINKED ISSUE #48: Update INPUT screenshots
With latest release, INPUT map canvas contains GPS panel and also, during recording, layers are now as floating button. 
Screenshot needs to be updated to new looks.

### LINKED ISSUE #194: Splitting geometries in Input
Input 1.6.0
https://github.com/MerginMaps/input/pull/2137

### LINKED ISSUE #195: Snapping in Input
Input 1.6.0
https://github.com/MerginMaps/input/pull/2126

### PR DESCRIPTION
**Pull request by alex-cit**
fix #195 added How to Set Up Snapping for Mergin Maps Input
fix #194 splitting geometries in Input added to https://merginmaps.com/docs/field/input_features/
fix #48 updated Input screenshots

### CODE REVIEW

**Review by alexbruy — APPROVED** [review]
Looks good to me. But I suggest asking @tomasMizera to take a look too, as he was working on the splitting stuff.

File: src/field/input_features.md [hunk 1]
```diff
@@ -39,15 +38,42 @@ Once you are finished, press **Done**. If your feature is an area, the shape wil
 
 ![Capturing lines](./input-capture-line.png)
 
-## Edit or delete features
+### Snapping features
+<Badge text="since Input 1.6.0" type="tip"/>
+
+Snapping can be enabled in your <MainPlatformName /> project to make the field survey easier. [How to Set Up Snapping](../gis/snapping/) will guide you through the snapping options.
+
+If snapping is enabled, the cross hairs will turn purple and snap to vertices (left) or segments (right) of existing features when capturing new features.
```
Comment (tomasMizera) [inline src/field/input_features.md:46]:
```suggestion
If snapping is enabled, the crosshair will turn purple and snap to vertices (left) or segments (right) of existing features when capturing new features.
```

File: src/gis/snapping/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+   
+   Don't forget to synchronise the project also in <MobileAppName /> before the fieldwork.
+
+Now you can use basic snapping in <MobileAppName />! 
+
+When capturing a new feature near an existing one, the cross hairs will turn purple and snap to its vertex (left) or to its segment (right).
+![project properties](./input_basic_snapping.png)
+
+:::tip
+If you don't want the cross hairs to snap to a feature, try zooming in. The snapping threshold is 20 pixels, so the more you zoom in, the closer you can place the vertex to the existing geometry without snapping.
```
Comment (tomasMizera) [inline src/gis/snapping/index.md:33]:
```suggestion
If you don't want the crosshair to snap to a feature, try zooming in. The snapping threshold is 20 pixels, so the more you zoom in, the closer you can place the vertex to the existing geometry without snapping.
```

File: src/field/input_features.md [hunk 3]
```diff
@@ -39,15 +38,42 @@ Once you are finished, press **Done**. If your feature is an area, the shape wil
...
+1. Tap on a feature and press the edit button.
+![Edit button](./input-edit-feature.png)
+
+2. Tap **Split geometry**
+![split geometry](./input-split-geometry.png)
+
+3. Create the splitting line by adding points. When finished, tap **Done**
+![split geometry](./input-split-geom-point.png)
+
+4. Two individual features are created!
```
Comment (tomasMizera) [inline src/field/input_features.md:70]: Well it can actually be more than two features, it depends on how many times the line crosses the geometry 🙂

**Review by tomasMizera — APPROVED** [review]
Looks great, just minor things, thanks!

**Pull request merged**